src/ucp/api/ucp.h

```diff
@@ -698,10 +698,12 @@ typedef enum {
                                                         synchronization with the
                                                         remote peer before releasing
                                                         the local send buffer */
-    UCP_OP_ATTR_FLAG_FORCE_IMM_CMPL = UCS_BIT(18)  /**< force immediate complete
+    UCP_OP_ATTR_FLAG_FORCE_IMM_CMPL = UCS_BIT(18), /**< force immediate complete
                                                         operation, fail if the
                                                         operation cannot be
```

Comment:
should the "fail..." part be documented here?  That's more to do with the behavior of code using this option, I think.

Reply:
How about 
"force immediate complete operation, fail and return UCS_ERR_NO_RESOURCE status if the operation cannot be completed immediately"
also since it's a fix for previous doc, ok to make it in a separate PR?

Reply:
Yeah, this should go to separate PR.